raft/raftpb/raft.proto

```diff
@@ -43,6 +43,7 @@ enum MessageType {
 	MsgHeartbeat     = 8;
 	MsgHeartbeatResp = 9;
 	MsgUnreachable   = 10;
+	MsgSnapStatus    = 11;
```

Comment:
Sounds good. If it's only sent on failure then maybe it's MsgSnapFailure (similar to MsgUnreachable) and we don't need the status/reject field.